custom_components/ha_washdata/diagnostics.py

```
@@ -76,4 +79,9 @@ async def async_get_config_entry_diagnostics(
             },
         },
         "store_export": _redact(exported),
+        # Rolling 24-hour in-memory buffers.
+        # power_trace:   [[iso_ts, watts], …] – every raw sensor reading
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)